CHANGELOG.md

```diff
@@ -42,6 +42,7 @@
 - Added ScaledObject Status Conditions to display status of scaling ([#750](https://github.com/kedacore/keda/pull/750))
 - Added optional authentication parameters for the Redis Scaler ([#962](https://github.com/kedacore/keda/pull/962))
 - Improved GCP PubSub Scaler performance by closing the client correctly ([#1087](https://github.com/kedacore/keda/pull/1087))
+- Enable support for authentication in metrics api scaler ([#1137](https://github.com/kedacore/keda/pull/1137))
```

Comment:
Since we are adding Metrics API Scaler in the same release (2.0), I wouldn't make a separate entry in the Changelog, but you can add a link to your PR to the extisting Metrics API Scaler entry

Reply:
yes, this makes sense. I missed the new section. Thanks. I have removed it.